sqlflow_models/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM sqlflow/modelzoo_base
+
+RUN pip install tensorflow==2.0.0b1 scikit-learn==0.20.0 numpy==1.16.2 pandas==0.25.1
+ADD *.py /sqlflow_models/
```

Comment:
why not `python setup.py install`? It should cover line 3 and line 4.

Reply:
The future model zoo only require a Dockerfile for each model collection, then model developers don't need to care about how to write a `setup.py`.